playbooks/common/openshift-cluster/upgrades/v3_3/master_config_upgrade.yml

```diff
@@ -48,3 +48,13 @@
     dest: "{{ openshift.common.config_base}}/master/master-config.yaml"
     yaml_key: 'controllerConfig.servicesServingCert.signer.keyFile'
     yaml_value: service-signer.key
+
+- modify_yaml:
+    dest: "{{ openshift.common.config_base}}/master/master-config.yaml"
+    yaml_key: 'admissionConfig.pluginConfig'
+    yaml_value: "{{ openshift.master.admission_plugin_config }}"
+
+- modify_yaml:
+    dest: "{{ openshift.common.config_base}}/master/master-config.yaml"
+    yaml_key: 'kubernetesMasterConfig.admissionConfig'
+    yaml_value:
```

Comment:
If this affects 3.3 clusters, then we should probably do it for 3.2 -> 3.3 upgrades. Wouldn't 3.3 systems that have this set already be broken prior to the 3.3 -> 3.4 upgrade?